src/generated/ec-time-off-service/AbsenceCountingMethod.ts

```diff
@@ -0,0 +1,303 @@
+/*
```

Comment:
I thought the same and remove these before but there was a problem. this generated client is mentioned in files like [here](https://github.com/SAP-samples/cloud-sdk-team-calendar/blob/ce3b1e89543f49cf2fdc0a72a085ebb781c7e8ad/src/read-appointments.ts#L7) that needed when a user runs `npm run watch:local` in [the first exercise](https://github.com/SAP-samples/cloud-sdk-team-calendar/blob/ce3b1e89543f49cf2fdc0a72a085ebb781c7e8ad/exercises/01-getting-started.md?plain=1#L79) before [the generated OData client step](https://github.com/SAP-samples/cloud-sdk-team-calendar/blob/ce3b1e89543f49cf2fdc0a72a085ebb781c7e8ad/exercises/02-generate-odata-clients.md?plain=1#L16). If the command is executed without them, compiler throws an error `Cannot find module './generated/MODULES`. That's because I keep the generated folder.